evap/staff/tests/test_views.py

```diff
@@ -1924,6 +1924,79 @@ def test_evaluation_copy(self):
         self.assertEqual(copied_evaluation.contributions.count(), 4)
 
 
+class TestEvaluationExamCreation(WebTestStaffMode):
+    csrf_checks = False
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.manager = make_manager()
+        cls.semester = baker.make(Semester)
+        cls.course = baker.make(Course, semester=cls.semester)
+        vote_start_datetime = datetime.datetime.now() - datetime.timedelta(days=50)
+        cls.evaluation = baker.make(Evaluation, course=cls.course, vote_start_datetime=vote_start_datetime)
+        cls.contributions = baker.make(
+            Contribution, evaluation=cls.evaluation, _fill_optional=["contributor"], _quantity=3, _bulk_create=True
+        )
+        cls.questionnaire = baker.make(Questionnaire, pk=83)
+        cls.url = reverse("staff:create_exam_evaluation")
+        cls.exam_date = (datetime.date.today()) + datetime.timedelta(days=10)
+
+    def test_create_exam_evaluation(self):
+        self.app.post(
+            self.url,
+            user=self.manager,
+            status=200,
+            params={"evaluation_id": self.evaluation.pk, "exam_date": self.exam_date},
+        )
+        self.assertEqual(Evaluation.objects.count(), 2)
+        exam_evaluation = Evaluation.objects.exclude(pk=self.evaluation.pk).get()
+        self.assertEqual(exam_evaluation.contributions.count(), self.evaluation.contributions.count())
+        self.assertEqual(
+            exam_evaluation.vote_start_datetime,
+            datetime.datetime.combine(self.exam_date + datetime.timedelta(days=1), datetime.time(8, 0)),
+        )
+        self.assertEqual(exam_evaluation.vote_end_date, self.exam_date + datetime.timedelta(days=3))
+        self.assertEqual(exam_evaluation.name_de, "Klausur")
+        self.assertEqual(exam_evaluation.name_en, "Exam")
+        self.assertEqual(exam_evaluation.course, self.evaluation.course)
+        self.assertEqual(exam_evaluation.participants.count(), self.evaluation.participants.count())
```

Comment:
`assertQuerySetEqual`?

Also, with the current test setup, I don't think the evaluation has any participants, does it?